va/va.go

```diff
@@ -893,6 +892,10 @@ func (va *ValidationAuthorityImpl) PerformValidation(ctx context.Context, domain
 			challenge.Status = core.StatusInvalid
 			challenge.Error = prob
 			logEvent.Error = prob.Error()
+			va.log.Info(fmt.Sprintf(
+				"Validation failed due to remote failures: identifier=%v err=%s",
+				authz.Identifier, prob))
+			va.metrics.remoteValidationFailures.Inc()
```

Comment:
Can you update `TestPerformRemoteValidation` to check for the log statement & counter increment?